ha-hoymiles-wifi-addon-nightly/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [v0.4]
```

Comment:
We haven't released 0.3 yet. 😀 I can fix the version number as part of the the next release. No need to rush.

Reply:
I think I need to do a version bump in the addon, otherwise home assistant won't show that an update exists, so I've been setting the addon versions independently of the releases